### ISSUE TITLE
Forward mouser scrolling event

### ISSUE THREAD

**Issue by zdm** [amcharts/amcharts5#1060]
Hi.

My chart is placed in the div which also has other content  and can be scrolled vertically.

Cart has `"wheelX": "panX"`. option
```
const chart = root.container.children.push( am5xy.XYChart.new( root, {
    "panX": true,
    "wheelX": "panX",
}
```

`whelY` option is not used.

When cursor is located on the chart and I am scrolling vertically - nothing happens, scroll event is ignored.
I am expecting that if mouse vertical scrolling event is not used by the chart - it will be forwarded to the top-level DOM element.

Is it possible to implement this?

**Comment by martynasma** [comment]
Fixed in [5.4.5](https://github.com/amcharts/amcharts5/releases/tag/5.4.5).

## [5.4.5] - 2023-08-26

### Added
- New method `addChildData()` added to `Hierarchy`. [More info](https://www.amcharts.com/docs/v5/charts/hierarchy/hierarchy-api/#Dynamically_adding_child_nodes).

### Fixed
- Vertical wheel-scrolling is no longer prevented if `wheelY: "none"` is set, regardless of the `wheelX` setting.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by zdm** [comment]
Thank you
